bin/docs.py

```diff
@@ -68,6 +68,10 @@
             if not details['edtf:deprecated'] == 'uuuu':
                 docs.write("* %s %s.\n" % ('This source was deprecated on', details['edtf:deprecated']))
 
+        #call out add date
+        if details.get('edtf:inception'):
+            docs.write("* %s %s.\n" % ('This source was added on', details['edtf:inception']))
```

Comment:
Please instead say:

- added: `YYYY-MM-DD`

Like:

`docs.write("* %s: `%s`\n" % ('added', details['edtf:inception']))`

Reply:
Also update deprecated to this same format, please.

Reply:
Say added, then deprecated.